types/src/stored_value.rs

```diff
@@ -806,71 +808,105 @@ mod serde_helpers {
         NamedKey(NamedKeyValue),
     }
 
-    impl<'a> From<&'a StoredValue> for BinarySerHelper<'a> {
+    pub struct InvalidHumanReadableDeser(String);
```

Comment:
Did you consider adding an `enum Error` in the `human_readable_compatibility` module to contain both this one and `NonUniqueContractVersion` (and possibly other variants)?

Reply:
no longer applicable